nix/pkgs/core.nix

```diff
@@ -14,19 +14,46 @@ rec {
     set -euxo pipefail
     export RUST_BACKTRACE=1
 
+    hc-standard-test
+    hc-slow-test
+    hc-wasm-test
+  '';
+
+  hcStandardTests = writeShellScriptBin "hc-standard-test" ''
+    set -euxo pipefail
+    export RUST_BACKTRACE=1
+
+    # limit parallel jobs to reduce memory consumption
+    export NUM_JOBS=8
+    export CARGO_BUILD_JOBS=8
+
+    # run all the non-slow cargo tests
+    cargo build --features 'build_wasms' -p holochain_wasm_test_utils
+    cargo test --workspace --exclude holochain --exclude release-automation --lib --tests --profile fast-test -- --nocapture
+  '';
+
+  hcSlowTests = writeShellScriptBin "hc-slow-test" ''
+    set -euxo pipefail
+    export RUST_BACKTRACE=1
+
     # limit parallel jobs to reduce memory consumption
     export NUM_JOBS=8
     export CARGO_BUILD_JOBS=8
 
     # alas, we cannot specify --features in the virtual workspace
     # run the specific slow tests in the holochain crate
-    cargo check --all-features --all-targets --manifest-path=crates/holochain/Cargo.toml
     cargo test --manifest-path=crates/holochain/Cargo.toml --features slow_tests,test_utils,build_wasms,db-encryption --profile fast-test -- --nocapture
-    # run all the remaining cargo tests
-    cargo check --all-features --all-targets --workspace --exclude holochain --exclude release-automation
-    cargo test --workspace --exclude holochain --exclude release-automation --lib --tests --profile fast-test -- --nocapture
+  '';
+
+  hcWasmTests = writeShellScriptBin "hc-wasm-test" ''
+    set -euxo pipefail
+    export RUST_BACKTRACE=1
+
+    # limit parallel jobs to reduce memory consumption
+    export NUM_JOBS=8
+    export CARGO_BUILD_JOBS=8
+
     # run all the wasm tests (within wasm) with the conductor mocked
-    cargo check --all-targets --lib --manifest-path=crates/test_utils/wasm/wasm_workspace/Cargo.toml --all-features
     cargo test --lib --manifest-path=crates/test_utils/wasm/wasm_workspace/Cargo.toml --all-features -- --nocapture
```

Comment:
how does it work when the `hc-wasm-test` is run separately from the `hc-standard-test`? specifically, how are the wasm files built, or are they not required by this?